dll/win32/shell32/folders/CDesktopFolder.h

```diff
@@ -145,4 +145,32 @@ class CDesktopFolder :
         END_COM_MAP()
 };
 
+class CDesktopFolderViewCB :
+    public CComObjectRootEx<CComMultiThreadModelNoCS>,
+    public IShellFolderViewCB,
+    public IFolderFilter
+{
+        CComPtr<IShellView> m_ShellView;
+        UINT8 m_IsProgmanHosted;
```

Comment:
```suggestion
        UINT8 m_IsProgmanHosted = 0u;
```
If our compilers support this of course,
then you can:
```cpp
CDesktopFolderViewCB() = default;
```